plasmapy/physics/distribution.py

```diff
@@ -9,14 +9,14 @@
 import numpy as np
 from scipy.special import gamma
 
-__all__ = [
-    "Maxwellian_1D",
-    "Maxwellian_velocity_3D",
-    "Maxwellian_speed_1D",
-    "Maxwellian_speed_3D",
-    "kappa_velocity_1D",
-    "kappa_velocity_3D",
-]
+__all__ = ["Maxwellian_1D",
+           "Maxwellian_velocity_2D",
+           "Maxwellian_velocity_3D",
+           "Maxwellian_speed_1D",
+           "Maxwellian_speed_2D",
+           "Maxwellian_speed_3D",
+           "kappa_velocity_1D",
+           "kappa_velocity_3D"]
```

Comment:
The reason I like having `]` in its own line and having every entry in a list like this ending with a comma is that when you add an entry, that shows in git as modifying one line instead of modifying 2-3.

Reply:
Ah, thanks for the tip. I'll keep that in mind for future modifications.